airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -1111,6 +1111,54 @@
     "public": true,
     "custom": false,
     "releaseStage": "alpha"
+  }, {
```

Comment:
Why is this file updated? Catalog updates should happen on merge.

Reply:
this was from ci 🤔 https://github.com/airbytehq/airbyte/commit/13100e008528c7a76759f9538251e026d62c61dd

(same with the connectors.md change)